From 0e9793f3d7abe113eb3917108a9899428882b68e Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 6 Aug 2013 18:20:44 +0100
Subject: [PATCH 043/102] sparsify: Advertize in --machine-readable that we
 support --check-tmpdir option.

This lets programs add the --check-tmpdir=[ignore|continue|fail]
option to avoid interactivity.

This fixes commit 7c463ac477168df5d4e4eb472ba01fa18b89c1a6.

(cherry picked from commit 5b278937dfd5177eaafcbc41830450ef5c0a2ab8)
---
 sparsify/sparsify.ml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sparsify/sparsify.ml b/sparsify/sparsify.ml
index fa5dd28..0364103 100644
--- a/sparsify/sparsify.ml
+++ b/sparsify/sparsify.ml
@@ -124,6 +124,7 @@ read the man page virt-sparsify(1).
     printf "virt-sparsify\n";
     printf "linux-swap\n";
     printf "zero\n";
+    printf "check-tmpdir\n";
     let g = new G.guestfs () in
     g#add_drive "/dev/null";
     g#launch ();
-- 
1.9.3

